packages/belos/src/BelosStatusTestGenResSubNorm.hpp

```diff
@@ -139,7 +139,8 @@ class StatusTestGenResSubNorm: public StatusTestResNorm<ScalarType,MV,OP> {
     </ul>
     </ol>
   */
-  int defineScaleForm( ScaleType TypeOfScaling, NormType TypeOfNorm, MagnitudeType ScaleValue = Teuchos::ScalarTraits<MagnitudeType>::one()) {
+  int defineScaleForm( ScaleType /* TypeOfScaling */, NormType /* TypeOfNorm */, MagnitudeType ScaleValue = Teuchos::ScalarTraits<MagnitudeType>::one()) {
```

Comment:
See above -- you can ignore `ScaleValue` but still give that parameter a default value:
`MagnitudeType /* ScaleValue */ = Teuchos::ScalarTraits<MagnitudeType>::one()`